### PR TITLE
Adding Image conversion link to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pal-16**
Just a suggestion to add the image conversion link for the leads to easily convert their image to the required format.

### CODE REVIEW

**Comment by pal-16** [comment]
Feel free to close this PR if the image link is not required. No worries!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,9 +35,11 @@ Import the image at the top of the file using:
 import abhinavsri360 from "../../photos/abhinavsri360.jpg";
 ```
 
+
 ### Notes for adding yourself to the map
 
 - Don't forget to add your `jpg` photo with resolution of `400px*400px` in `src/photos/` with the file name as your GitHub username and import it.
+- Please feel free to use https://resizeimage.net/ to get your image to the required resolution with the `jpg` format in one go!
```
Comment (aravindvnair99) [inline CONTRIBUTING.md:42]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,9 +35,11 @@ Import the image at the top of the file using:
 import abhinavsri360 from "../../photos/abhinavsri360.jpg";
 ```
 
+
 ### Notes for adding yourself to the map
 
 - Don't forget to add your `jpg` photo with resolution of `400px*400px` in `src/photos/` with the file name as your GitHub username and import it.
```
Comment (aravindvnair99) [inline CONTRIBUTING.md:41]:
```suggestion
- Don't forget to add your `jpg` photo with a resolution of `400px*400px` in `src/photos/` with the file name as your GitHub username and import it. Use a tool such as [this](https://resizeimage.net) to achieve the same.
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@pal-16 That's a good addition, better to keep as one point though.

**Review by aravindvnair99 — APPROVED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Yes, looks good to me.

**Pull request merged**